pkg/network/server.go

```diff
@@ -849,6 +860,20 @@ func (s *Server) handleMPTDataCmd(p Peer, data *payload.MPTData) error {
 	return s.stateSync.AddMPTNodes(data.Nodes)
 }
 
+// requestMPTNodes requests specified MPT nodes from the peer or broadcasts
+// request if peer is not specified.
+func (s *Server) requestMPTNodes(p Peer, itms []util.Uint256) error {
+	if len(itms) == 0 {
+		return nil
+	}
+	if len(itms) > payload.MaxMPTHashesCount {
+		itms = itms[:payload.MaxMPTHashesCount]
+	}
```

Comment:
Shouldn't we request multiple batches in this case?

Reply:
Actually, such situation is impossible given the current implementation, because `GetUnknownMPTNodesBatch` is restricted by `MaxMPTHashesCount`. See also https://github.com/nspcc-dev/neo-go/pull/2019#discussion_r695148487.